src/main/java/com/minecolonies/coremod/inventory/api/CombinedItemHandler.java

```diff
@@ -262,6 +262,21 @@ public ItemStack extractItem(final int slot, final int amount, final boolean sim
         return null;
     }
 
+    @Override
+    public int getSlotLimit(final int slot)
+    {
+        int slotIndex = slot;
+        for (final IItemHandlerModifiable modifiable : handlers)
+        {
+            if (slotIndex >= modifiable.getSlots())
+                slotIndex-=modifiable.getSlots();
+            else
+                return modifiable.getSlotLimit(slotIndex);
+        }
+
+        return 0;
+    }
+
     protected IItemHandlerModifiable[] getHandlers()
     {
         return handlers;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Return a copy of "handlers". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2384)